tests/e2e/metrics/handlers/basic_handler.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+from aws_lambda_powertools import Metrics
+from aws_lambda_powertools.metrics import MetricUnit
+
+METRIC_NAMESPACE = os.environ["METRIC_NAMESPACE"]
+METRIC_NAME = os.environ["METRIC_NAME"]
+SERVICE_NAME = os.environ["SERVICE_NAME"]
+
+metrics = Metrics(namespace=METRIC_NAMESPACE, service=SERVICE_NAME)
```

Comment:
both are optional if you already are setting environment variable - just make sure the env var follows our standard: `POWERTOOLS_METRICS_NAMESPACE` and `POWERTOOLS_SERVICE_NAME`.

then it becomes `metrics = Metrics()`

Reply:
ok